mp/src/game/shared/momentum/weapon/weapon_mom_rocketlauncher.cpp

```diff
@@ -104,7 +169,21 @@ void CMomentumRocketLauncher::RocketLauncherFire()
     pPlayer->m_iShotsFired++;
 
     DoFireEffects();
-    WeaponSound(SINGLE);
+
+#ifdef CLIENT_DLL
+    static ConVarRef mom_rj_sounds("mom_rj_sounds");
+#endif
+
+    switch (mom_rj_sounds.GetInt())
+    {
+    case 0:
+        break;
+    case 1:
+        WeaponSound(SINGLE);
+        break;
+    case 2:
+        WeaponSound(SPECIAL1);
+    }
```

Comment:
A `switch` statement that only has two actual cases can just be an if / else if check:

```
if (mom_rj_sounds.GetInt() == 1)
{
    WeaponSound(SINGLE);
}
else if (mom_rj_sounds.GetInt() == 2)
{
    WeaponSound(SPECIAL1);
}
```